### DISCUSSION TITLE
Timestamps accuracy on long audios

### DISCUSSION THREAD

**Discussion by despairTK** [Purfview/whisper-standalone-win#161] [General]
It's not a serious issue, but the timestamps for transcribing short audio are more accurate than the timestamps for long audio. The timestamp of long audio may be displayed in advance or delayed. This situation generally occurs in some very short sentences. Is there any way to make the timestamp of long audio more accurate?

**Comment by Purfview** [comment]
Is it transcription, not translation? How much "advance or delayed"?

Share all output files and an audio sample where it's inaccurate, preferably in European language.

**Reply by despairTK** [reply]
It's a transcription.
Okay, I will collect the audio files where this problem occurs. In fact, this situation will occur in every language audio, mainly depending on the duration of the transcribed audio.

**Comment by Purfview** [comment]
Check if you have this issue with `--condition_on_previous_text=False -prompt=None`
